code/modules/hivemind/objects.dm

```diff
@@ -3,20 +3,24 @@
 
 //toxic shot, turret's ability use it
 /obj/item/projectile/goo
-	name = "electrolyzed goo"
+	name = "Electrolyzed goo"
 	icon = 'icons/obj/hivemind.dmi'
 	icon_state = "goo_proj"
 	damage_types = list(BURN = 15)
-	check_armour = ARMOR_ENERGY //Unlike Bio, it's not either 0% or 100%. Strong Energy armour isn't common, But most of armour has some protection against energy.
+	check_armour = ARMOR_ENERGY
 	step_delay = 2
 
+/obj/item/projectile/goo/weak
+	name = "Weakened Electrolyzed goo"
+	damage_types = list(BURN = 5)
+
 
 /obj/item/projectile/goo/on_hit(atom/target)
 	. = ..()
 	if( isliving(target) && !issilicon(target) )
 		var/mob/living/L = target
-		L.damage_through_armor(15, TOX, attack_flag = ARMOR_RAD)
+		L.damage_through_armor(5, TOX, attack_flag = ARMOR_RAD)
```

Comment:
With this damage and something like a technomancer voidsuit the projectiles would do no damage with GDR.

Reply:
should I increase it (either back to 15 or a different value) then?

Reply:
I'd bump it up to 10 and change the armor type to energy.

Reply:
Done so